pkg/test/testdata/testdata/cortex/config.yaml

```diff
@@ -1,115 +0,0 @@
-# Configuration for running Cortex in single-process mode.
-# This should not be used in production.  It is only for getting started
-# and development.
-
-auth_enabled: true
-tenant_federation:
-  enabled: true
-
-server:
-  log_level: warn
-  http_listen_port: {{ .HttpListenPort }}
-  grpc_listen_port: {{ .GrpcListenPort }}
-  http_tls_config: &tls_server_config
-    cert_file: "{{ .StorageDir }}/server.crt"
-    key_file: "{{ .StorageDir }}/server.key"
-    client_ca_file: "{{ .StorageDir }}/root.crt"
-    client_auth_type: RequireAndVerifyClientCert
-  grpc_tls_config: *tls_server_config
-
-alertmanager:
-  alertmanager_client: &tls_client_config
-    tls_enabled: true
-    tls_cert_path: "{{ .StorageDir }}/client.crt"
-    tls_key_path: "{{ .StorageDir }}/client.key"
-    tls_ca_path: "{{ .StorageDir }}/root.crt"
-    tls_server_name: localhost
-
-querier:
-  store_gateway_client: *tls_client_config
-  active_query_tracker_dir: "{{ .StorageDir }}/active-query-tracker"
-
-frontend:
-  grpc_client_config: *tls_client_config
-
-frontend_worker:
-  grpc_client_config: *tls_client_config
-  match_max_concurrent: true
-
-ingester_client:
-  grpc_client_config: *tls_client_config
-
-ruler:
-  evaluation_interval: 1s
-  poll_interval: 1s
-  resend_delay: 1s
```

Comment:
these values are important to keep as default for the test environment, so wherever these are set now we need to make sure they are the same

Reply:
They are set here https://github.com/rancher/opni/blob/ff9ffbdb415585a1debe0b3268a358147c32f3ae/plugins/metrics/test/test_drivers.go#L271-L276

Reply:
Looks like it's missing ResendDelay though so I'll add that